peercrawler.py

```diff
@@ -257,8 +257,8 @@ def crawl_peer(peer: Peer):
             try:
                 logger.debug("Query %39s:%5s (score:%4s)" % ('[%s]' % p.ip, p.port, p.score))
                 self.add_peers(peer, self.get_peers_from_peer(peer))
-            except Exception as e:
-                logger.error('Unexpected exception in crawl_peer()\n' + traceback.format_exc())
+            except:
```

Comment:
@miguel1117 it is not advisable to catch all exceptions because that catches the kill signal and Control-C.
The general advise is to catch Exception exceptions only.

Reply:
I see, I wasn't aware of that.